AdminUi/apps/admin_ui/lib/pages/login_page.dart

```diff
@@ -17,6 +17,10 @@ class _LoginScreenState extends State<LoginScreen> {
   final _apiKeyController = TextEditingController();
   final _apiKeyFocusNode = FocusNode();
 
+  bool _isButtonEnabled = false;
+  bool apiKeyValid = false;
```

Comment:
Why is this public?

Reply:
I don't know. I definitely forgot to add _ to variable name.

Reply:
I forgot to put an underscore to make it private.